scripts/build-meteor.sh

```diff
@@ -19,7 +19,7 @@ meteor npm install
 # build the bundle
 printf "\n[-] Building Meteor application...\n\n"
 mkdir -p $APP_BUNDLE_DIR
-meteor build --directory $APP_BUNDLE_DIR
+meteor build --server-only --directory $APP_BUNDLE_DIR
```

Comment:
I'm not too sure, but I thought it would have the exact same effect of removing the platforms prior to running the build, so it shouldn't affect people with no mobile apps.  The reason you would build the mobile is to generate the .apk file which would not be relevant for deploying the server.